src/Functions/UserDefined/UserDefinedSQLObjectsZooKeeperStorage.cpp

```diff
@@ -406,7 +406,7 @@ void UserDefinedSQLObjectsZooKeeperStorage::syncObjects(const zkutil::ZooKeeperP
     LOG_DEBUG(log, "Syncing user-defined {} objects", object_type);
     Strings object_names = getObjectNamesAndSetWatch(zookeeper, object_type);
 
-    getLock();
+    auto lock = getLock();
```

Comment:
```suggestion
    auto lock = getLock();
    (void)lock;
```

Reply:
Builds are OK, I think it is not necessary